### PR TITLE
Pauls branch

### LINKED ISSUE #177: My code is perfect R is clearly the issue
* **Is your feature request related to a problem? Please describe.**

I have written beautiful code, R is the problem. 

* **Describe the solution you'd like**
@caityrice Fix R

* **Describe alternatives you've considered**
Throwing laptop out window
Shouting at screen
* **Additional context**

### PR DESCRIPTION
**Pull request by Paul-McEvoy**
## Proposed changes

Fix R to solve this

## Types of changes

solve R
-
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x ] I have checked that my changes have not broken any other functionality
- [x ] I have linked to any issues this PR fixes 

## Who is reviewing this PR?

@mirandamathieson

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -1 +1,10 @@
-This is a random text file. You can add and remove things to it to test how git works.
\ No newline at end of file
+git branchThis is a random text file. You can add and remove things to it to test how git works.
+test test
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:2]: Did you need to repeat this?

File: text_file.txt [hunk 2]
```diff
@@ -1 +1,10 @@
...
+<<<<<<< HEAD
+
+
+ALL YOUR TEXT ARE BELONG TO US
+=======
+Adding to Paul's file
+testing
+>>>>>>> c1e5a70f92e42fd84b4973e7df18bb79ccf5dfd3
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:10]: This looks ugly

**Review by DIPAD-Fran-Bryden — COMMENTED** [review]
Looks fine!

File: text_file.txt (hunk 1 above)
Comment (mirandamathieson) [inline text_file.txt:2]: I like the duplication of test!

**Review by mirandamathieson — COMMENTED** [review]
Thanks for letting me review this! :)

**Review by mirandamathieson — APPROVED** [review]

**Pull request closed without merging**